src/js/session_report.js

```diff
@@ -94,6 +96,18 @@ export class SessionReport {
     get iceConnectionsLost() {
         return this._iceConnectionsLost;
     }
+    /**
+     * Tells if the RTCSession has failed ICE connection in talking state.
+     */
+    get iceConnectionsFailed() {
+        return this._iceConnectionsFailed;
+    }
+    /**
+     * Tells if the RTCSession has failed Peer connection in talking state.
+     */
+    get connectionFailed() {
```

Comment:
Can we rename this to PeerConnectionFailed and rest of the places as well? Just to be more descriptive.

Reply:
Will address this